src/drivers/sdr/sdr_uart.c

```diff
@@ -86,6 +90,21 @@ void csp_sdr_rx(void *cb_data, void *buf, size_t len, void *pxTaskWoken) {
             csp_queue_enqueue(ifdata->rx_queue, ifdata->rx_mpdu, QUEUE_NO_WAIT);
         }
     }
+
```

Comment:
Did you run this code? It just seems like it was put in a strange place

Reply:
yes